conf/helmfile.d/0100.redis.yaml

```diff
@@ -30,11 +31,22 @@ releases:
 
         cluster:
           enabled: true
-          slaveCount: 2
+          slaveCount: 3
 
         sentinel:
           enabled: true
 
+        configmap: |-
```

Comment:
I'm not sure how to tell what commit this came from. Can you explain why we have these keys in the configmap?

Reply:
These changes are from #316. We were having a redis issue where the master would fall into a restart loop during high-volume loads with the following error:

```
Asynchronous AOF fsync is taking too long (disk is busy?). Writing the AOF buffer without waiting for fsync to complete, this may slow down Redis.
```

From the PR:
> The redis config file says something about using no-appendfsync-on-rewrite yes if latency is an issue.

Reply:
Other changes to the configmap were also introduced for latency issues in high volume clusters.